src/styles/sidebar/sidebar.scss

```diff
@@ -15,6 +15,9 @@ $base-line-height: 20px;
 @import './components/dropdown-menu-btn';
 @import './components/excerpt';
 @import './components/group-list';
+@import './components/group-section';
+@import './components/group';
+@import './components/group-out-of-scope';
```

Comment:
A small request - could you keep this list sorted please? It makes it more obvious where to slot new values into place in future.